tools/benchmark.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# Performs benchmark, and append data to //gh-pages/data.json.
+
+import os
+import sys
+import json
+import time
+from util import run, run_output, root_path, build_path
+
+benchmark_types = ["hello", "relative_import"]
+benchmark_files = ["tests/002_hello.ts", "tests/003_relative_import.ts"]
+
+data_file = "gh-pages/data.json"
+benchmark_file = "benchmark.json"
+
+
+def read_json(filename):
+    with open(filename) as json_file:
+        return json.load(json_file)
+
+
+def write_json(filename, data):
+    with open(filename, 'w') as outfile:
+        json.dump(data, outfile)
+
+
+def prepare_gh_pages_dir():
+    if os.path.exists("gh-pages"):
+        return
+    try:
+        run([
+            "git", "clone", "--depth", "1", "-b", "gh-pages",
+            "https://github.com/denoland/deno.git", "gh-pages"
+        ])
+    except:
+        os.mkdir("gh-pages")
+        with open("gh-pages/data.json", "w") as f:
+            f.write("[]")  # writes empty json data
+
+
+def main(argv):
+    if len(argv) == 2:
+        build_dir = sys.argv[1]
+    elif len(argv) == 1:
+        build_dir = build_path()
+    else:
+        print "Usage: tools/benchmark.py [build_dir]"
+        sys.exit(1)
+
+    os.chdir(root_path)
+    prepare_gh_pages_dir()
+    run(["hyperfine", "--export-json", benchmark_file, "--warmup", "3"] + [
+        os.path.join(build_dir, "deno") + " " + file
+        for file in benchmark_files
+    ])
```

Comment:
It would be nice to be able to specify arguments. Like I want to run the same script with —reload and without.